### ISSUE TITLE
[XAML bug] The x to close the "Pin Widget" window is unresponsive when over titlebar

### ISSUE THREAD

**Issue by mathewleeza** [microsoft/devhome#537]
### Steps to reproduce the bug

Description
Clicked Add Widget in the dashboard page. Attempted to close the window and it would not.

**Comment by krschau** [comment]
This is a big problem on screens that are the height of the dialog, see #746

**Comment by krschau** [comment]
This is a known limitation of custom titlebars in WinUI 3.

**Comment by Phlosioneer** [comment]
Ran into this today, had to end-task the program. I'm not sure how github's issue search works exactly, but this issue didn't appear for the search "is:issue is:open close button". Almost submitted a duplicate ticket. I notice that the word "button" is not anywhere on this issue, so that might have been the reason. I don't know if this comment fixes the search, or if the word "button" should be added to the issue description, to avoid duplicate tickets.

**Comment by cinnamon-msft** [comment]
This appears to be fixed! Please let us know if you're still hitting this issue
